packages/ui/src/components/Loading/LoadingErrorWithSecondaryAction.tsx

```diff
@@ -0,0 +1,59 @@
+import ExclaimationMarkCircle from "@litespace/assets/ExclaimationMarkCircle";
+import { Button } from "@litespace/ui/Button";
+import { useFormatMessage } from "@/hooks/intl";
+import { Typography } from "@litespace/ui/Typography";
+import { Void } from "@litespace/types";
+import React from "react";
+import cn from "classnames";
+
+export const LoadingErrorWithSecondaryAction: React.FC<{
+  retry: Void;
+  error: string;
+  secondAction: {
+    label: string;
+    onClick: Void;
+  };
+  size?: "small" | "medium" | "large";
+}> = ({ retry, error, secondAction, size = "small" }) => {
+  const intl = useFormatMessage();
+
+  return (
+    <div className="tw-flex tw-flex-col tw-items-center tw-justify-center">
+      <div
+        className={cn(
+          "tw-flex tw-items-center tw-justify-center tw-bg-destructive-200 tw-rounded-full",
+          {
+            "tw-p-[3.33px] tw-w-10 tw-h-10": size === "small",
+            "tw-p-[5.33px] tw-w-16 tw-h-16": size === "medium",
+            "tw-p-[6.67px] tw-w-20 tw-h-20": size === "large",
```

Comment:
No need to have sizes and variants. This component is only used once in the verify email page.